src/openproduct/producten/urls.py

```diff
@@ -14,7 +14,23 @@
 description = f"""
 Een API voor Producten.
 
+De Producten API heeft 1 endpoint `producten`. Hiermee kunnen producten zelf, de bijbehorende eigenaren en documenten worden aangemaakt.
+
+## Uitleg per resource
+
+### Product
 Een product is een instantie van een producttype (zie producttypen API), in een product worden onder andere de gegevens van de eigenaar, de benodigde data voor het product en bijvoorbeeld de status vastgelegd.
+Specifieke product data kan worden opgeslagen in de JSON velden `dataobject` en `verbruiksobject`. Deze velden worden gevalideerd door de `verbruiksobject_schema` & `dataobject_schema` velden van het producttype ([zie jsonschema](https://json-schema.org)).
+De status van een product kan alleen worden veranderd naar de een van de `toegestane statussen` gedefineerd op het producttype.
+
+### Eigenaar
+Aan een product kunnen één of meerdere eigenaren worden gelinkt. Een eigenaar kan een Partij (Open Klant), natuurlijk of niet natuurlijke persoon zijn.
```

Comment:
Any supplier can implement this API, it doesn't have to be Open Klant I think
```suggestion
Aan een product kunnen één of meerdere eigenaren worden gelinkt. Een eigenaar kan een Partij (Klantinteracties API), natuurlijk of niet natuurlijke persoon zijn.
```